src/encoded/static/components/filegallery.js

```diff
@@ -3035,17 +3035,19 @@ class FileGalleryRendererComponent extends React.Component {
         if (!this.props.altFilterDefault) {
             this.setState({ selectedFilterValue: '0' });
         }
+
         // Determine how many visualizable files there are
-        // const tempFiles = filterForVisualizableFiles(this.state.files);
-        // If the graph is hidden and there are no visualizable files, set default tab to be table and set default assembly to be 'All assemblies'
-        // if (this.props.hideGraph && tempFiles.length < 1) {
-        if (this.props.hideGraph) {
-            this.setState({ currentTab: 'tables' }, () => {
-                this.filterFiles('All assemblies', 'assembly');
+        const vizFiles = filterForVisualizableFiles(this.props.context.files);
+
+        if (!this.props.hideGraph && vizFiles.length > 0) {
+            this.setState({ currentTab: 'browser' }, () => {
+                // Determine available assemblies
+                const assemblyList = this.setAssemblyList(this.state.files);
```

Comment:
This part uses `this.state.files` while the `vizFiles` above uses `this.props.context.files`. Is this correct? I’m unclear when you should use one or the other.